### ISSUE TITLE
Allow reading of status bar in Microsoft Outlook with Office 365

### ISSUE THREAD

**Issue by rob-aph** [nvaccess/nvda#13669]
### Is your feature request related to a problem? Please describe.
When attempting to read the status bar in Outlook, the message, "No status line found" is always given.
### Describe the solution you'd like
The status bar should be read, as with other supported applications.
### Describe alternatives you've considered
I modified the outlook app module to solve this problem, and am willing to submit a pull request after discussion. I am running Outlook 16.0.15128.20178 from Office 365, and don't have access to other versions for testing.

**Comment by josephsl** [comment]
Hi, have you tried after maximizing Outlook window? When I perform read status bar command on my system, status bar content such as message count is announced (I’m using the same Outlook build, too). Thanks.

**Comment by rob-aph** [comment]
Yes, I am running Outlook maximized, but your question caused me to try with the Window restored, and it does work when restored. Very strange, but I see now that the command is working for some, and possibly most, users. When I restored, the window only got slightly smaller.

It’s also strange, because I have the same problem on two different machines. I’ll keep poking around and close this issue if I get more clues.

**Comment by lukaszgo1** [comment]
@rob-aph Have you set your taskbar to be automatically hidden by any chance?

**Comment by rob-aph** [comment]
Yes, I am hiding my Windows task bar when it is not in use. I'll try not hiding it and see if anything changes.

**Comment by rob-aph** [comment]
I disabled hiding of the Windows task bar, and the Outlook status bar reads fine, whether Outlook is restored or maximized. Interesting! I wonder what would be involved to fix NVDA to read the status bar for those who hide their status bar and run with Outlook maximized.

**Comment by lukaszgo1** [comment]
This affects not only Outlook but all applications which window occupies the entire screen. I've encountered this in my add-on for Becky! Internet Mail, and since I believe I understand the problem well enough  I'll submit a fix when time permits.

**Comment by rob-aph** [comment]
Thank you Lukasz. I was going to take a crack at it, but it sounds like you are much closer to a solution than I.

**Comment by Adriani90** [comment]
This works as expected with NVDA 2025.3.1 and MS outlook 365 version 2509. Closing as works for me.

**Comment by CyrilleB79** [comment]
@Adriani90 do you confirm that you have tested with the same setup as @rob-aph or @lukaszgo1 before closing as "work-for-me"? I.e. taskbar automatically hidden and Outlook window maximized.

**Comment by Adriani90** [comment]
Yes I can confirm that.
